### ISSUE TITLE
getting banned has excessive console messages

### ISSUE THREAD

**Issue by Tsunder** [Regalis11/Barotrauma#9863]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

getting banned results in various console spam from the sequence disconnect, referencing file names and lines of code

expectation is for getting banned to be handled more gracefully

### Reproduction steps

1. be a client
2. be mid-round
3. get banned
4. open F3 

### Bug prevalence

Happens every time I play

### Version

0.18.15.1 / 0.18.15.2 on macOS

### Which operating system did you encounter this bug on?

Windows

**Comment by gerakis** [comment]
"happens every time i play" hahahahaha

**Comment by Regalis11** [comment]
I'm afraid I could not reproduce this. The only message I get is the notification about getting disconnected, which I think is useful information.
![image](https://user-images.githubusercontent.com/14351070/189839730-ebf06a8e-bcd4-4756-ae1e-611b5d1c8522.png)

Could you post a screenshot of the exact messages you get when you're banned?

**Comment by Tsunder** [comment]
hmm, ok, could not reproduce either.

the incident that led me to report this was that there was also a "attempting to reconnect" dialogue shortly (<5s) before getting the banned message.

FWIW it doesn't actaully happen every time i've played, i don't get banned that much. it has actually happened only once so far.

**Comment by Regalis11** [comment]
Closing as a possible duplicate of https://github.com/Regalis11/Barotrauma/issues/9849
